### PR TITLE
Allow updating the metadata block size after filesystem creation

### PR DESCRIPTION
**Pull request by acolombier**
This is related to [this feature request](https://groups.google.com/g/s3ql/c/nCSg6_Tuz0g).

### CODE REVIEW

File: src/s3ql/database.py [hunk 1]
```diff
@@ -664,6 +667,33 @@ def next_dirty_block():
         if not update_params:
             return
 
+        if flush_old_seq_no:
```
Comment (Nikratio) [inline src/s3ql/database.py:670]: Could you add a unit test that checks if this works as intended?

File: .vscode/settings.json
[the remark this answers is not in the document]
Comment (acolombier) [reply .vscode/settings.json:1]: Sure thing, I can move it to another PR as well. It looks like depending which extension you use for Python, `python.formatting.blackArgs` may not be supported and `black-formatter.*` appears to be the alternative.

File: src/s3ql/database.py [hunk 2]
```diff
@@ -615,13 +615,16 @@ def upload_metadata(
     params: FsAttributes,
     incremental: bool = True,
     update_params: bool = True,
+    flush_old_seq_no: bool = False,
```
[the remark this answers is not in the document]
Comment (acolombier) [reply src/s3ql/database.py:618]: I will move that in an other function

**Comment by Nikratio** [comment]
Are you still interested in working on this? Otherwise I'll close the pull request for now.

**Comment by acolombier** [comment]
I am indeed, but haven't had much time to work on the requested changes, and not sure when I will have some. Is it okay to keep as draft? otherwise, feel free to close it

**Comment by acolombier** [comment]
Closing for now as I am lacking time to get this done. Hopefully will find the time to get that done in the future, but if anyone came across it and gets the chance to finish it, please do!

**Pull request closed without merging**